### ISSUE TITLE
Wood doesn't Grind

### ISSUE THREAD

**Issue by DumbDave961** [minetest-mods/technic#623]
Although the grinding recipes for grinding group:tree and group:wood are registered, the trees and planks won't output common wood grindings. In fact, the grinders only grind the rubber trees into rubber tree grindings.

In my personal experience I have found that machine recipes don't accept groups for inputs. I'm not sure how normal crafting recipes use them.

**Comment by OgelGames** [comment]
Yeah, technic machines don't support groups, they never have. A workaround is to add recipes for every wood type.

Group support should be coming to [technic plus](https://github.com/mt-mods/technic) soon, as I've been working on it recently.

**Comment by DumbDave961** [comment]
Yes, that's what I had to do for Technic Elepower Compatibility. I'm glad to hear Technic Plus will be able to use groups. Would you mind explaining how that works? I'm curious, and DustyBagel is working on mods with fixes for Technic and Elepower.

**Comment by DustyBagel** [comment]
So there are no plans to make technic machine recipes use groups or is that currently in development?

**Comment by SmallJoker** [comment]
@DustyBagel There is currently no-one actively taking care of minetest-mods/technic, which is probably the reason why the spin-off mt-mods/technic exists. Everyone is kindly invited to propose PRs to improve this mod. I will review and test those before merging.

**Comment by DustyBagel** [comment]
@SmallJoker Oh, I see. Well then maybe I will take a look for myself and see if I can get a pull request together some time.#
